cobalt/docker/webtest/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG BASE_IMAGE_SHA=sha256:4abf773f2a570e6873259c4e3ba16de6c6268fb571fd46ec80be7c67822823b3
+
+FROM debian:12@${BASE_IMAGE_SHA}
```

Comment:
Is there a reason why we are not pulling debian image from marketplace.gcr.io/google/debian12, just like we do in the linux docker file?

Reply:
This was copied from the unittest image file. I guess it wasn't updated with this fix.

Reply:
Fixed unittest and webtest Dockerfiles